src/redux/modules/notifications.js

```diff
@@ -0,0 +1,49 @@
+// action types
+export const CREATE_NOTIFICATION = 'whatever/notifications/CREATE_NOTIFICATION';
+const DELETE_NOTIFICATION = 'whatever/notifications/DELETE_NOTIFICATION';
+
+// initial state
+const initialState = {
+  items: [],
+};
+
+// action creators
+export const createNotification = notification => dispatch => {
```

Comment:
We depend on having an `id` property on the `notification` instance. It means that the responsibility to provide a correct `notification` object is on the caller side, which is not really safe. Since there are some mandatory values in that object, best thing you could do is to provide a default new `notification` in this function, and just override its properties with the object from function arguments.